### DISCUSSION TITLE
（β版？）エディタの表示切り替えアイコンマウスオーバー時のヘルプのポップアップが欲しい

### DISCUSSION THREAD

**Discussion by simon-ritchie** [increments/qiita-discussions#254] [要望・アイデア・不具合]
※β版のエディタを使っているので恐らくβ版のアップデート関係だと思いますが、もし以前のアップデートですでに更新済みで見落としていて勘違いしていたら恐縮です！

![image](https://user-images.githubusercontent.com/25178045/179868902-170095c0-9454-400c-9320-f7f2ce4d3f16.png)

エディタ右上のエディタだけを表示したり左右分割したりプレビューを表示したりのアイコンが右上に配置・アップデートになったと思いますが（確か以前まで矢印のアイコン？）、本日記事を書いていたら「あれ、エディタのみの表示のアイコンどこにいったのだろう・・・」と思い、「右上のアイコン・・・なのだろうか・・・？」と一瞬迷ったり試しに各アイコン押したりとしておりました。

この辺に関しまして押さないとなんのアイコンなのか判断が付きづらいため、押す前にマウスオーバー時等になんの挙動をするアイコンなのかヘルプのポップアップが表示されるとユーザーにとって分かりやすいかもしれません！

※隣の絵文字やヘルプのアイコンではマウスオーバー時にヘルプのポップアップが出るので同じようなものをイメージしております！

![image](https://user-images.githubusercontent.com/25178045/179869489-aa8441f7-df8e-4cc0-8a80-530eec3b3b9d.png)

**Comment by tanaka-asami415** [comment]
@simon-ritchie 
ありがとうございます！こちら前向きに検討させていただきます💪✨
他にもお気づきな点がございましたら、フィードバックをお待ちしております！！🙇

**Comment by ohakutsu** [comment]
@simon-ritchie 
おまたせしました :bow: 

こちら対応完了しました！
https://qiita.com/release-notes#2022-08-08

**Reply by simon-ritchie** [reply]
@tanaka-asami415 
@ohakutsu 

おお、ご対応ありがとうございます・・・！🙇‍♂️
